### ISSUE TITLE
Achilles tendon tear, partial or complete

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11159]
**Preferred term label:**
Achilles tendon tear, partial or complete

**Synonyms**

**Definition (free text, please give PubMed ID)**
MRI of the foot/ankle showing partial or complete tear of the achilles tendon

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Achilles tendon rupture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001433
- label: Achilles tendon tear
- parents: [Abnormality of the Achilles tendon (HP:0005109)](https://hpo.jax.org/app/browse/term/HP:0005109)
- pmid: 
- def: A partial or complete break in the Achilles tendon (the tendon that connects the gastrocnemius and soleus to the calcaneus). This feature can be ascertained by magnetic resonance imaing (MRI) of the foot/ankle showing partial or complete tear of the achilles tendon.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11159
- synonyms:

**Comment by pnrobinson** [comment]
done
